core/src/main/java/greencity/configuration/SecurityConfig.java

```diff
@@ -154,7 +154,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 UBS_LINK + "/ubs/order/{id}/cancellation/",
                 ADMIN_LINK + "/**",
                 "/accountinfo")
-            .hasAnyRole(ADMIN)
+            .hasAnyRole(ADMIN, UBS_EMPLOYEE)
```

Comment:
is this config correct?
should `UBS_EMPLOYEE` has access to `SUPER_ADMIN_LINK + "/**"` and  `ADMIN_LINK + "/**"`?